.nvmrc

```diff
@@ -1 +1 @@
-v10.18.1
+lts/dubnium
```

Comment:
@whymarrh tagging you on this as you were the latest to update. Are we good with this to support all versions of node 10 with nvm (whichever version of 10 the dev has installed with nvm)? I tried finding a way to do `^v10.18.1` but came up empty.

Reply:
We do need to ensure that v10.12.0 at minimum is used, because we use the `recursive` flag for `mkdir` in a few places (which was added in `v10.12.0`). That's why this repo is more specific than just `v10`.

But it looks like specifying `lts/dubnium` ensures the latest version is always used? That sounds convenient! I didn't know that.

Could you submit this change as a separate PR? I like the idea of always asking for the latest, so we don't need to keep bumping this. At least until we get v12 working anyway.

Reply:
I think if the dev only had v10.11.x it would use that rather than forcing them to install the latest, so I think we might wanna hold on changing it at atm. I'll drop this change from the PR and open a separate for discussion.

Reply:
Hmm - I tried it locally and `v10` would use the latest v10.x.x I had installed locally, but `lts/dubnium` forced me to install the latest. But I don't know how `nvm` works, or how it knows what the latest is :thinking: (maybe it checks each time, or periodically? :man_shrugging: ). I don't see any clear documentation about this behaviour.